common/djangoapps/student/tests/test_email.py

```diff
@@ -90,33 +91,51 @@ def append_allowed_hosts(self, hostname):
         self.addCleanup(settings.ALLOWED_HOSTS.pop)
 
 
+@ddt.ddt
 @unittest.skipUnless(settings.ROOT_URLCONF == 'lms.urls', 'Test only valid in lms')
-class ActivationEmailTests(CacheIsolationTestCase):
+class ActivationEmailTests(EmailTemplateTagMixin, CacheIsolationTestCase):
     """
     Test sending of the activation email.
     """
 
-    ACTIVATION_SUBJECT = u"Action Required: Activate your {} account".format(settings.PLATFORM_NAME)
+    ACTIVATION_SUBJECT = u"Action Required: Activate your {} account".format(
```

Comment:
@shadinaif the pattern `configuration_helpers.get_value('PLATFORM_NAME', settings.PLATFORM_NAME)` is only useful on in production code. Test code should use just one of them:

 - Either `configuration_helpers.get_value('PLATFORM_NAME')`
 - Or `settings.PLATFORM_NAME`

Please apply for all other items below.

Reply:
Sorry ! fixed